daemon/main.c

```diff
@@ -2070,6 +2068,16 @@ int main(int argc, char **argv) {
     if(become_daemon(dont_fork, user) == -1)
         fatal("Cannot daemonize myself.");
 
+    // The "HOME" env var points to the root's home dir because Netdata starts as root. Can't use "HOME".
+    struct passwd *pw = getpwuid(getuid());
```

Comment:
I see that [failure to switch to the user is not fatal](https://github.com/netdata/netdata/blob/1bad78f37806aeddb9a2b1147360c9dc75b27d8d/daemon/daemon.c#L506-L515), that is why switched to `getpwuid(getuid())`.